opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -485,6 +486,8 @@ class WellModelGhostLastMatrixAdapter : public Dune::AssembledLinearOperator<M,X
         }
 
         bool solve(Vector& x) {
+            const int verbosity = prm_.get<int>("verbosity");
```

Comment:
Do we know that the `verbosity` parameter always exists?  If not, that could be a possible reason for the very weird [regression failures](https://ci.opm-project.org/job/opm-simulators-PR-builder/969/testReport/) we're getting.

Reply:
It should not be the cause, but it could trigger it:
- verbosity might be an arbitrary value, as prm_ is empty for defautl run.
- Assuming it is higher than ten, the (not 100% tested?) `Opm::Helper::writeSystem()` will be called
- If that overwrites something, then it might explain the problems.

This pure speculation as of now, though. I can be wrong and often have been.

Reply:
I am able to reproduce locally and will look into this.

Reply:
Its me (of course :angry:). This is the throwing version. Hence it throws an exception, the time stepper chops the step (a different bug),  and here we are...